LabManagement/PowerShell/JITUserQuota/readme.md

```diff
@@ -1,21 +1,20 @@
-Setup JIT User Quota with Managed Identity
 
-- Setup Automation account with runbook using Managed identity.
-    - https://docs.microsoft.com/azure/automation/learn/powershell-runbook-managed-identity
-- Setup modules
-    - Add Az.LabServices.BulkOperations.psm1 as Az.LabServices.BulkOperations.zip
-- Add RunBook PowerShell code
-    - Copy the JITQuotaWithManagedId.ps1 into the runbook editor
-- Publish
-- Set schedule
--
-Setup JIT User Quota with RunAs
+# Just-in-time User Quota
 
-- Setup Automation account with runbook using RunAs.
-    - https://docs.microsoft.com/azure/automation/manage-runas-account
-- Setup modules
-    - Add Az.LabServices.BulkOperations.psm1 as Az.LabServices.BulkOperations.zip
-- Add RunBook PowerShell code
-    - Copy the JITQuotaWithRunAs.ps1 into the runbook editor
-- Publish
-- Set schedule
\ No newline at end of file
+Sample that creates a just-in-time system to update a lab user's quota as needed.
+
+## Setup with Managed Identity
+
+- **Setup Automation account with runbook using Managed identity**.  See [Tutorial: Create Automation PowerShell runbook using managed identity](https://docs.microsoft.com/azure/automation/learn/powershell-runbook-managed-identity) for details
+- **Setup modules**.  Add Az.LabServices.BulkOperations.psm1 as Az.LabServices.BulkOperations.zip
+- **Add Runbook PowerShell code**.  Copy the JITQuotaWithManagedId.ps1 into the runbook editor
+- **Publish to Runbook**.  See [managing runbooks](https://learn.microsoft.com/azure/automation/manage-runbooks#publish-a-runbook) for details.
+- **Set schedule**.  See [managing runbooks](https://learn.microsoft.com/azure/automation/manage-runbooks#schedule-a-runbook-in-the-azure-portal) for details.
+
+## Setup with RunAs

```

Comment:
RunAs is being deprecated - we can probably just remove this part from the readme.    https://learn.microsoft.com/en-us/azure/automation/migrate-run-as-accounts-managed-identity?tabs=sa-managed-identity

Reply:
Good catch.  Section removed and warning added to script.